testsuite/tests/uid-deps/link_intf_impl.reference

```diff
@@ -2,17 +2,16 @@
 Uid dependencies:
 Link_intf_impl.0 <- Link_intf_impl.1
 Link_intf_impl.1 <- Link_intf_impl.0
-Link_intf_impl.4 <- Link_intf_impl.2
+Link_intf_impl.4 <-> Link_intf_impl.2
```

Comment:
This commit that refactors mark and directionality together appears to change the testsuite output. This is a bit surprising, is it expected? Is it an improvement?

Reply:
It was indeed a regression, fixed in [045312f](https://github.com/ocaml/ocaml/pull/13308/commits/045312f184dd5c4e11d66ad2f022f873c12af9ea)